packages/tmux/tmux.pacscript

```diff
@@ -0,0 +1,23 @@
+name="tmux"
+version="3.2a"
+url="https://github.com/tmux/tmux/releases/download/${version}/tmux-${version}.tar.gz"
+build_depends="gcc make pkg-config bison"
+depends="libncurses-dev libevent-dev"
+breaks="tmux-git"
+replace="tmux"
+description="A terminal multiplexer"
+hash="551553a4f82beaa8dadc9256800bcc284d7c000081e47aa6ecbb6ff36eacd05f"
+optdepends=("libutempter-dev: Record user sessions to utmp and wtmp files [add to depends array and rebuild to enable]")
```

Comment:
```suggestion
optdepends=("libutempter-dev: Record user sessions to utmp and wtmp files")
```

In version `1.6` (currently `develop`) the `optdepends` are called before the installation begins, just like `depends`, so no need to give this warning. The dependency will be available during install.

Reply:
Fixed in https://github.com/pacstall/pacstall-programs/pull/361/commits/615b145c09a3c6bd509325f5cba8e977f1e09e50